ArchiSteamFarm/Core/ASF.cs

```diff
@@ -73,6 +73,8 @@ public static class ASF {
 
 	internal static readonly SemaphoreSlim OpenConnectionsSemaphore = new(WebBrowser.MaxConnections, WebBrowser.MaxConnections);
 
+	internal static string DebugDirectory => Path.Combine(SharedInfo.DebugDirectory, OS.ProcessStartTime.ToString("yyyy-MM-dd-THH-mm-ss", CultureInfo.InvariantCulture));
```

Comment:
I don't believe we need to be second-accurate

Reply:
Of course we do, two program restarts are enough to cause duplicate debugging files generated.